Jenkinsfile

```diff
@@ -4,18 +4,31 @@ pipeline {
     }
 
     stages {
+        stage('Install Python dependencies') {
+            steps {
+                echo 'Installing Python dependencies'
+                sh 'python3 -m venv .venv'
+                sh '''. .venv/bin/activate
+                python3 -m pip install -U -r requirements.txt
+                '''
+            }
+        }
         stage('Setup') {
             steps {
-                sh 'cp /usr/local/etc/roms/baserom_oot.z64 baserom_original.z64'
-                sh 'make -j setup'
+                sh 'cp /usr/local/etc/roms/baserom_oot.z64 baseroms/gc-eu-mq-dbg/baserom.z64'
+                sh '''. .venv/bin/activate
+                make -j setup
+                '''
             }
         }
         stage('Build (qemu-irix)') {
             when {
                 branch 'main'
             }
             steps {
-                sh 'ORIG_COMPILER=1 make -j'
+                sh '''. .venv/bin/activate
+                make -j ORIG_COMPILER=1
```

Comment:
Cool I guess #1599 broke Jenkins building with irix-qemu ido :) nice catch

Reply:
Yeah, that was totally my intention and not me just preferring to pass makefile variable at the right instead of the left. Totally

:sweat_smile: